<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestScreen</class>
 <widget class="QWidget" name="TestScreen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="drawStack" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>440</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>230</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowBack">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>btnNext</zorder>
  <zorder>textBrowser</zorder>
  <zorder>btnShowBack</zorder>
  <zorder>btnPrev</zorder>
  <zorder>line</zorder>
  <zorder>openGLWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawStack</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">drawstack.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
